db/db.select/main.c

```diff
@@ -302,6 +499,23 @@ void parse_command_line(int argc, char **argv)
         G_fatal_error(
             _("You must provide one of these options: <%s>, <%s>, or <%s>"),
             sql->key, input->key, table->key);
+    // Check for mutually exclusive options
+    if (parms.format == JSON) {
+        fatal_error_option_value_excludes_option(
+            format, fs, _("Separator is part of the format"));
+        fatal_error_option_value_excludes_option(
+            format, vs, _("Vertical separator is part of the format"));
+        fatal_error_option_value_excludes_option(
+            format, nv, _("Null value is part of the format"));
+    }
+    if (v->answer) {
+        G_verbose_message(
+            _("Flag 'v' is deprecated and will be removed in a future "
+              "release. Please use format=vertical instead [deprecated]."));
+        if (format->answer && parms.format != VERTICAL) {
```

Comment:
What I meant to was putting the [deprecated] at the end of the description on line 420.